src/App.tsx

```diff
@@ -10,9 +10,20 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="home" element={<Navigate to="/" replace />} />
+
+            <Route path="people">
```

Comment:
It would be better to use the `index` prop on the main `Route` for the 'people' path instead of creating a nested `Route` with the `index` prop.

Reply:
not sure that I get it correct, but changed